tools/cmake/set_linker_script.cmake

```
@@ -35,10 +35,11 @@ function(mbed_set_linker_script input_target raw_linker_script_path)
         set(LinkerScriptTarget ${input_target}LinkerScript)
         add_custom_target(${LinkerScriptTarget} DEPENDS ${LINKER_SCRIPT_PATH} VERBATIM)
         add_dependencies(${input_target} ${LinkerScriptTarget})
+        # We hardcore mmemory file to "application"
```

```suggestion
        # We hardcode mmemory file to "application"
```

Though hardcoring sounds like a lot of fun ;)

---

=) fixing now, thanks